riff-raff/app/assets/javascripts/notifications.coffee

```diff
@@ -0,0 +1,17 @@
+Notification.requestPermission()
+
+intervalId = null
+
+checkStatus = () ->
+  if $('[data-run-state]').length != 0
+    buildName = window.riffraff.buildName
+    switch $('[data-run-state]').data('run-state')
+      when 'Failed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' failed!'})
+      when 'Completed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' is finished'})
+    disableCheck()
+
+if $('[data-run-state]').length == 0
+  intervalId = setInterval checkStatus, 800
```

Comment:
Rather than set your own timer you might be able to use `postRefresh`. See https://github.com/guardian/riff-raff/blob/master/riff-raff/app/assets/javascripts/stop-deploy.coffee#L11-L14